R-packages/evalcast/R/get_predictions.R

```diff
@@ -57,14 +62,6 @@ get_predictions <- function(forecaster,
 
 #' Get predictions cards for a single date
 #'
-#' @template forecaster-template
-#' @template name_of_forecaster-template
-#' @template signals-template
-#' @template forecast_date-template
-#' @template incidence_period-template
-#' @template ahead-template
-#' @template geo_type-template
-#' @template geo_value-template
```

Comment:
These lines are deleted because the function is not exported?

Reply:
We should keep these so that R CMD check doesn't complain.

Reply:
Done. Mea culpa.